### ISSUE TITLE
Wrong BatchNorm2d momentum value in ONNX export

### ISSUE THREAD

**Issue by nerox8664** [pytorch/pytorch#18525]
## 🐛 Bug

Wrong BatchNorm2d momentum value in ONNX export. Whatever momentum is set in PyTorch, it's always `1.0` in ONNX export.

## To Reproduce

1. Create PyTorch model:
```
import torch
import torch.nn as nn
import numpy as np
from torch.autograd import Variable

class PytorchModel(nn.Module):
    def __init__(self):
        super(PytorchModel, self).__init__()
        self.bn = nn.BatchNorm2d(1, momentum=0.5)

    def forward(self, x):
        x = self.bn(x)
        return x

pytorch_model = PytorchModel()
pytorch_model.eval()

x_data = np.random.uniform(0, 1, size=(1, 1, 224, 224))
x = Variable(torch.Tensor(x_data))
```

2. Convert it to ONNX:
```
torch_out = torch.onnx._export(pytorch_model, x, "model.onnx", export_params=True)
```

3. Load ONNX model:
```
import onnx
import caffe2.python.onnx.backend as onnx_caffe2_backend
model = onnx.load("model.onnx")

model
```

```
attribute {
      name: "momentum"
      f: 1.0
      type: FLOAT
    }
```

## Expected behavior

BatchNorm2d's `momentum` should have proper value.

## Environment

```
 - PyTorch Version (e.g., 1.0): 1.0
 - OS (e.g., Linux): Arch Linux
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.6
 - CUDA/cuDNN version: -
 - GPU models and configuration: -
 - Any other relevant information: -

**Comment by soumith** [comment]
cc: @houseroad @zrphercule

**Comment by spandantiwari** [comment]
@nerox8664 and @houseroad - I can repro this on my side. I am taking a look.

**Comment by spandantiwari** [comment]
@nerox8664 and @houseroad - the fix has been merged. We can close this issue.

**Comment by houseroad** [comment]
Thanks a lot for fixing this @spandantiwari
